lib/diagnostic.rb

```diff
@@ -43,6 +55,9 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
-  end
+    lines = filename.open(ARGV[0]) do |file|
+      file.map(&:itself)
+    end
+
+  print lines
```

Comment:
There's no need to reference `ARGV[0]` here; since our first parameter is named, we can refer to it by name.